tests/core/timeseries/test_io.py

```diff
@@ -795,3 +795,77 @@ def test_export_influxdb2_tidy(settings_si_false):
             df = request.values.all().df
             ExportMixin(df=df).to_target("influxdb2://orga:token@localhost/?database=dwd&table=weather")
             mock_connect.assert_called_once_with(url="http://localhost:8086", org="orga", token="token")  # noqa: S106
+
+
+@surrogate("influxdb_client_3.InfluxDBClient3")
+@surrogate("influxdb_client_3.Point")
+@surrogate("influxdb_client_3.WriteOptions")
+@surrogate("influxdb_client_3.write_client_options")
+@surrogate("influxdb_client_3.write_client.client.write_api.WriteType")
+@pytest.mark.remote
+def test_export_influxdb3_tabular(settings_si_false):
+    """Test export of DataFrame to influxdb v3"""
+    request = DwdObservationRequest(
+        parameter=DwdObservationDataset.CLIMATE_SUMMARY,
+        resolution=DwdObservationResolution.DAILY,
+        period=DwdObservationPeriod.RECENT,
+        settings=settings_si_false,
+    ).filter_by_station_id(station_id=[1048])
+    with mock.patch(
+        "influxdb_client_3.InfluxDBClient3",
+    ) as mock_client, mock.patch(
+        "influxdb_client_3.Point",
+    ), mock.patch(
+        "influxdb_client_3.WriteOptions",
+    ) as mock_write_options, mock.patch(
+        "influxdb_client_3.write_client_options",
+    ) as mock_write_client_options, mock.patch(
+        "influxdb_client_3.write_client.client.write_api.WriteType",
+    ) as mock_write_type:
```

Comment:
Is it Python syntax for a cascaded context manager client, without needing to do to the same level of indentations? This is sweet, I didn't know about it.

Reply:
I mean the mocks are equal of importance right?

Reply:
I think all code is of equal importance, based on its importance. Here, I was referring to the compact, list-based notation of driving multiple context managers, in which case they are nested, as if written with nested `with` statements, but do not take so much screen space when writing them down.